pkg/repository/v1/sqlcv1/triggers.sql

```diff
@@ -26,8 +30,8 @@ JOIN
     "WorkflowTriggers" as triggers ON triggers."workflowVersionId" = latest_versions."workflowVersionId"
 JOIN
     "WorkflowTriggerEventRef" as eventRef ON eventRef."parentId" = triggers."id"
-WHERE
-    eventRef."eventKey" = ANY(@eventKeys::text[]);
+JOIN event_keys k ON k.event_key LIKE REPLACE(eventRef."eventKey", '*', '%')
```

Comment:
This will do a sequential scan over the `WorkflowTriggerEventRef`, but I think this is fine since there won't be very many rows here AFAICT